pkg/agent/utils.go

```diff
@@ -468,13 +468,9 @@ func setCustomKubeletConfig(customKc *datamodel.CustomKubeletConfig,
 		}
 		if customKc.CPUCfsQuotaPeriod != "" {
 			kubeletConfig.CPUCFSQuotaPeriod = datamodel.Duration(customKc.CPUCfsQuotaPeriod)
-			// enable CustomCPUCFSQuotaPeriod feature gate is required for this configuration.
-			kubeletConfig.FeatureGates["CustomCPUCFSQuotaPeriod"] = true
 		}
 		if customKc.TopologyManagerPolicy != "" {
 			kubeletConfig.TopologyManagerPolicy = customKc.TopologyManagerPolicy
-			// enable TopologyManager feature gate is required for this configuration.
```

Comment:
Is this feature gate only removed in 1.29? Do we still need 
`kubeletConfig.FeatureGates["TopologyManager"] = true` for other versions? 

If yes, we cannot merge and release this before corresponding aks-rp version is out.

Reply:
Ah just saw the comments above, looks like yes.